subsys/pcd/Kconfig

```diff
@@ -6,27 +6,54 @@
 
 menu "Peripheral CPU DFU (PCD)"
 
-menuconfig PCD
-	bool "Peripheral CPU DFU Support"
-	default y if SOC_NRF5340_CPUNET && IS_SECURE_BOOTLOADER
+config PCD
+	bool "Peripheral CPU DFU Support [DEPRECATED]"
+	select PCD_APP
+	select PCD_NET
+
+if PCD
+	comment "The PCD option is deprecated, please use PCD_APP or PCD_NET instead"
+endif
+
+config PCD_APP
+	bool "Peripheral CPU DFU Support for Application Processor"
 	default y if SOC_SERIES_NRF53X && MCUBOOT && BT_RPMSG_NRF53
+
+config PCD_NET
+	bool "Peripheral CPU DFU Support for Network Processor"
+	default y if SOC_NRF5340_CPUNET && IS_SECURE_BOOTLOADER
 	select FLASH_PAGE_LAYOUT
 	select STREAM_FLASH
 	select STREAM_FLASH_ERASE
 
-if PCD
+config PCD_USE_CONSTANTS
+	bool "Use KConfig constants rather than pm_config.h"
 
-config PCD_BUF_SIZE
-	int
-	default 512
-	help
-	  Must be <= the page size of the flash device.
+config PCD_CMD_ADDRESS
+	hex "PCD Command Address in RAM"
+	depends on PCD_USE_CONSTANTS
+
+config NET_CORE_APP_OFFSET
+	hex "Network Core Application Offset in Flash"
+	depends on PCD_USE_CONSTANTS
+
+if PCD_APP || PCD_NET
```

Comment:
Shouldn't this enclose also `config PCD_USE_CONSTANTS` (this option does not seem to be useful without `PCD_APP` or `PCD_NET` enabled)?

Reply:
Move that and those depending on it